### ISSUE TITLE
importing "github.com/spf13/cobra" panics

### ISSUE THREAD

**Issue by fiws** [containous/yaegi#257]
The yaegi repl panics when i try to import "github.com/spf13/cobra"

Not sure if i am doing anything wrong, as yaegi does not seem to find any other libs i throw at it. (always getting "unable to find source related to). But it should not panic.

Expected result:
```console
 yaegi                        
> import "github.com/spf13/cobra"

```

Got:
```console
 yaegi                        
> import "github.com/spf13/cobra"
panic: reflect: FieldByName of non-struct type

goroutine 1 [running]:
reflect.(*rtype).FieldByName(0xc000097da0, 0xc0002c61db, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/usr/lib/go/src/reflect/type.go:942 +0x135
github.com/containous/yaegi/interp.(*itype).lookupBinField(0xc0003a69c0, 0xc0002c61db, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/fiws/go/src/github.com/containous/yaegi/interp/type.go:602 +0xa9
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc0002e7900)
	/home/fiws/go/src/github.com/containous/yaegi/interp/cfg.go:1160 +0x399e
github.com/containous/yaegi/interp.(*node).Walk(0xc0002e7900, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:113 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc0002e7800, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002e7600, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002e7500, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002e6200, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002b3800, 0xc000192f70, 0xc000192fa8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc000096180, 0xc0002b3800, 0x1, 0x16, 0x0, 0x0, 0x5)
	/home/fiws/go/src/github.com/containous/yaegi/interp/cfg.go:39 +0x1b2
github.com/containous/yaegi/interp.(*Interpreter).importSrcFile(0xc000096180, 0x0, 0x0, 0xc0001522c1, 0x16, 0xc0001522d2, 0x5, 0x0, 0xc000193308)
	/home/fiws/go/src/github.com/containous/yaegi/interp/src.go:76 +0x3b8
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc000237000, 0xc000193610)
	/home/fiws/go/src/github.com/containous/yaegi/interp/gta.go:141 +0xc5e
github.com/containous/yaegi/interp.(*node).Walk(0xc000237000, 0xc000193610, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc000236300, 0xc000193610, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000236100, 0xc000193610, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc000096180, 0xc000236100, 0xc000152ca0, 0x16, 0xc000152ca0, 0x16)
	/home/fiws/go/src/github.com/containous/yaegi/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).importSrcFile(0xc000096180, 0x0, 0x0, 0xc000152061, 0x16, 0xc000152072, 0x5, 0x0, 0x8)
	/home/fiws/go/src/github.com/containous/yaegi/interp/src.go:68 +0xade
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc0001ba300, 0xc000155ee0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/gta.go:141 +0xc5e
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ba300, 0xc000193c70, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ba200, 0xc000193c70, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0001ba000, 0xc000193c70, 0x0)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc000096180, 0xc0001ba000, 0xc000148070, 0x4, 0x0, 0xc000148070)
	/home/fiws/go/src/github.com/containous/yaegi/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc000096180, 0xc000152020, 0x20, 0xc000193df0, 0x1f, 0xd14415, 0x1, 0xc000152020)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:252 +0x104
github.com/containous/yaegi/interp.(*Interpreter).Repl(0xc000096180, 0xc000010010, 0xc000010018)
	/home/fiws/go/src/github.com/containous/yaegi/interp/interp.go:332 +0x1c3
main.main()
	/home/fiws/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:85 +0x4e2

```

**Comment by mvertes** [comment]
Thanks for the report. You didn't do anything wrong. The panic occurs during the evaluation of the source of cobra package or one of its dependency. Need to investigate!

**Comment by mvertes** [comment]
Fixed by 058c121273fcbbf83460e442e6e74b1de7610866
